nebula-components/components/_navbar.scss

```diff
@@ -6,6 +6,13 @@ $nb-navbar-background-color: $nb-color-white !default;
 $nb-navbar-toggle-color: $nb-color-black !default;
 $nb-navbar-toggle-background: $nb-color-white !default;
 $nb-navbar-collapsed-width: 350px !default;
+$nb-content-padding-top-sticky: $nb-navbar-height !default;
+
+@if $nb-navbar-sticky {
```

Comment:
We should actually create a component called `c-site-content` and put the `if` block inside of that, the navbar shouldn't know about `site-content`.  You can also just use the `$nb-navbar-height` variable in there rather than creating an alias in here.
e.g.

```
.#{$nb-namespace}c-site-content {
  @if $nb-navbar-sticky {
    padding-top: $nb-navbar-height-collapsed;

    @include nb-respond-to('nav') {
      padding-top: $nb-navbar-height;  
    }
  }
}
```

We can also loose the modifier `c-site-content--padding-top` the consumer shouldn't need to add it, it will just be automated this way if `$nb-navbar-sticky` is set to `true`.

Give me a shout if you require further explanation.